# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsFrameLoader.cpp
# Commit: 707a48ef2338
# Full Hash: 707a48ef23385b4b143a9218fe788efcd3e1e3fc
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2019-05-23 15:55:11
# Regressor Bug: 1525720
# File Overlap Count: 1
# Description:
#   Bug 1525720, part 3 - Remove method to createRemoteFrameLoader from nsIMozBrowserFrame interface. r=farre
#   
#   This appears unused and adds unneeded surface area for these API's to support.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31431
# ==============================================================================

diff -r 37fb118dac6c -r 707a48ef2338 dom/base/nsFrameLoader.cpp
--- a/dom/base/nsFrameLoader.cpp	Wed Apr 24 23:05:40 2019 -0500
+++ b/dom/base/nsFrameLoader.cpp	Wed Apr 24 23:25:56 2019 -0500
@@ -3043,10 +3043,10 @@
   return do_AddRef(mOwnerContent);
 }
 
-void nsFrameLoader::SetRemoteBrowser(nsIRemoteTab* aBrowserParent) {
+void nsFrameLoader::InitializeFromBrowserParent(BrowserParent* aBrowserParent) {
   MOZ_ASSERT(!mBrowserParent);
   mIsRemoteFrame = true;
-  mBrowserParent = BrowserParent::GetFrom(aBrowserParent);
+  mBrowserParent = aBrowserParent;
   mChildID = mBrowserParent ? mBrowserParent->Manager()->ChildID() : 0;
   MaybeUpdatePrimaryBrowserParent(eBrowserParentChanged);
   ReallyLoadFrameScripts();